### PR TITLE
Update Linkerd Governance

### PR DESCRIPTION
**Pull request by olix0r**
The governance structure documented is `GOVERNANCE.md` is no longer
suitable for the project and doesn't reflect the reality of how changes
are made.

This change proposes an updated, simplified governance structure that
clearly outlines the expectations for maintainers around project
participation and decision making. It is expected that *most*
contributions will not come from maintainers; but we need a core group
of maintainers that are ultimately responsible for technical stewardship
of the project.

### CODE REVIEW

**Review by wmorgan — APPROVED** [review]

**Review by adleong — APPROVED** [review]
🏛️

File: GOVERNANCE.md [hunk 1]
```diff
@@ -2,46 +2,57 @@
...
+Linkerd is for everyone. Anyone can become a Linkerd contributor simply by
+contributing to the project, whether through code, documentation, blog posts,
+community management, or other means. As with all Linkerd community members,
+contributors are expected to follow the [Linkerd Code of
+Conduct][coc].
 
-## Expectations
+All contributions to Linkerd code, documentation, or other components in the
+Linkerd GitHub org must follow the guidelines in [CONTRIBUTING.md][contrib].
+Whether these contributions are merged into the project is the perogative of
```
Comment (alpeb) [inline GOVERNANCE.md:15]: prerogative

**Review by alpeb — APPROVED** [review]

**Review by kleimkuhler — APPROVED** [review]
👍

File: GOVERNANCE.md [hunk 2]
```diff
@@ -2,46 +2,57 @@
...
+Linkerd is for everyone. Anyone can become a Linkerd contributor simply by
+contributing to the project, whether through code, documentation, blog posts,
+community management, or other means. As with all Linkerd community members,
+contributors are expected to follow the [Linkerd Code of
+Conduct][coc].
 
-## Expectations
+All contributions to Linkerd code, documentation, or other components in the
+Linkerd GitHub org must follow the guidelines in [CONTRIBUTING.md][contrib].
+Whether these contributions are merged into the project is the prerogative of
```
Comment (hawkw) [inline GOVERNANCE.md:15]: typo:
```suggestion
Whether these contributions are merged into the project is the perogative of
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -2,46 +2,57 @@
...
 
-Maintainers are responsible for one or more components, and are expected to
-contribute code, field incoming PRs, triage issues, proactively fix bugs, and
-generally perform maintenance tasks for these components.
+## Maintainer Expectations
 
-Super-maintainers are responsible for the project as a whole, and are expected
-to guide general project direction as well as being the final reviewer on PRs.
+Maintainers have the ability to merge code into the project. Anyone can
+become a Linkerd maintainer (see "Becoming a maintainer" below.)
```
Comment (hawkw) [inline GOVERNANCE.md:21]: nit: maybe "Becoming a maintainer" should be a link here?

**Review by hawkw — APPROVED** [review]
looks good to me overall; couple  minor suggestions.

File: GOVERNANCE.md (hunk 2 above)
Comment (olix0r) [reply GOVERNANCE.md:15]: https://github.com/linkerd/linkerd2/pull/5477#pullrequestreview-562066340 -- turns out it's actually prerogative.

Comment (hawkw) [reply GOVERNANCE.md:15]: whoa, i guess i've been spelling it wrong my entire life. never mind, then!

**Review by klingerf — APPROVED** [review]

**Review by siggy — APPROVED** [review]
👍 🚢

**Review by zaharidichev — APPROVED** [review]

**Review by rmars — APPROVED** [review]
🌟 Ⓜ️ 🔍

**Comment by olix0r** [comment]
@hemakl I'd love to have unanimous approval on this PR. Do you have time to review these governance changes?

**Comment by hemakl** [comment]
🚢 👍

**Review by hemakl — APPROVED** [review]

**Comment by olix0r** [comment]
Thanks, everyone!

**Pull request merged**